src/components/com_tjdashboard/administrator/sql/install.mysql.utf8.sql

```diff
@@ -15,20 +15,13 @@ CREATE TABLE IF NOT EXISTS `#__tj_dashboards` (
   `modified_by` int(10) UNSIGNED NOT NULL DEFAULT '0',
   `context` varchar(255) NOT NULL COMMENT 'Dashboard context',
   `parent` int(11) NOT NULL COMMENT 'dashboard id of parent dashboard',
-  PRIMARY KEY (`dashboard_id`)
+  PRIMARY KEY (`dashboard_id`),
+  KEY `idx_checkout` (`checked_out`),

```

Comment:
No need of a key on checked_out and state. 

alias should also be a unique key.